osbenchmark/aggregator.py

```diff
@@ -193,24 +193,27 @@ def build_aggregated_results(self):
 
     def calculate_weighted_average(self, task_metrics: Dict[str, List[Any]], iterations: int) -> Dict[str, Any]:
         weighted_metrics = {}
+        num_executions = len(next(iter(task_metrics.values())))
+        total_iterations = iterations * num_executions
```

Comment:
To clarify, `iterations` is the number of iterations that the user inputted in workload params / default number of times a task is run in a workload? Not the number of times we executed the same test?

Reply:
That's correct!